modules/administration-guide/partials/proc_restoring-a-che-instance-from-its-latest-backup-using-prod-cli.adoc

```diff
@@ -0,0 +1,42 @@
+[id="restoring-a-{prod-id-short}-instance-from-its-latest-backup-using-{prod-cli}_{context}"]
+= Restoring a {prod-short} instance from its latest backup using {prod-cli}
+
+To restore the {prod-short} instance from its latest backup using `{prod-cli}`, you can enter `latest` as the snapshot ID.
+
+include::snip_warning-backup-snapshots-are-bound-to-specific-cluster.adoc[]
+
+.Prerequisites
+
+* Installed `{prod-cli}`.
+* Backup snapshot of a {prod-short} instance on any of the following:
+** xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[External SFTP backup server]
+** xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3]
+** xref:backups-of-che-instances-to-a-rest-backup-server.adoc[External REST backup server]
+** xref:backups-of-che-instances-to-the-internal-backup-server.adoc[Internal backup server]
+
+.Procedure
+
+. Run the `{prod-cli} server:restore` command with the following arguments:
++
+[subs="+quotes,+attributes"]
+----
+$ {prod-cli} server:restore \
+--snapshot-id=latest \ <1>
+--repository-url=_<repository-url>_ \ <2>
+--repository-password=_<repository-password>_ <3>
+----
+<1> Specifies the latest snapshot.
+<2> For the external backup server: the backup repository URL as an argument using the `--repository-url` or `-r` option. (Omitted when using the internal backup server.)
+<3> For the external backup server: the backup repository password as an argument using the `--repository-password` or `-p` option. (Omitted when using the internal backup server.)
++
+include::snip_tip-omitting-prod-cli-options-for-server-backup-and-server-restore-onward-when-using-same-backup-server.adoc[]
+
+. When prompted in the output, enter `y` to confirm:
```

Comment:
You may add a tip, that it is possible to skip this question by passing `--batch` flag (especially useful for scripts)
The same for other occurrences.